pilot/pkg/config/kube/gateway/controller.go

```diff
@@ -286,13 +400,6 @@ func (c *Controller) Delete(typ config.GroupVersionKind, name, namespace string,
 }
 
 func (c *Controller) RegisterEventHandler(typ config.GroupVersionKind, handler model.EventHandler) {
```

Comment:
I'm guessing this is still around to fulfill an interface? Does nothing register an event handler for the gateway controller? Should we wire it up anyway